packages/components/src/internal/dropdowns.tsx

```diff
@@ -342,6 +342,7 @@ export const MenuItem = forwardRef<HTMLLIElement, MenuItemProps>((props, ref) =>
         },
         [disabled, href, onClick]
     );
+    // TODO: Use AppLink (needs support for role and rel)
```

Comment:
Marker for TODO.

Reply:
I'd agree there is merit in just having `AppLink` extend all the properties of an anchor tag.

Reply:
I plan to do this in a separate PR for now